test/e2e/plain_provisioner_test.go

```diff
@@ -1105,24 +1098,18 @@ var _ = Describe("plain provisioner bundleinstance", func() {
 
 			// ensure the original BI owns the underlying bundle before creating the duplicate
 			By("projecting a successful installation status for the original BundleInstance")
-			Eventually(func() bool {
+			Eventually(func() (*metav1.Condition, error) {
 				if err := c.Get(ctx, client.ObjectKeyFromObject(biOriginal), biOriginal); err != nil {
-					return false
-				}
-				if biOriginal.Status.InstalledBundleName != bundle.GetName() {
-					return false
-				}
-				existing := meta.FindStatusCondition(biOriginal.Status.Conditions, rukpakv1alpha1.TypeInstalled)
-				if existing == nil {
-					return false
-				}
-				expected := metav1.Condition{
-					Type:   rukpakv1alpha1.TypeInstalled,
-					Status: metav1.ConditionStatus(corev1.ConditionTrue),
-					Reason: rukpakv1alpha1.ReasonInstallationSucceeded,
+					return nil, err
 				}
-				return conditionsSemanticallyEqual(*existing, expected)
-			}).Should(BeTrue())
+				return meta.FindStatusCondition(biOriginal.Status.Conditions, rukpakv1alpha1.TypeInstalled), nil
+			}).Should(And(
+				Not(BeNil()),
+				WithTransform(func(c *metav1.Condition) string { return c.Type }, Equal(rukpakv1alpha1.TypeInstalled)),
+				WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionFalse)),
```

Comment:
To match the original logic?
```suggestion
				WithTransform(func(c *metav1.Condition) metav1.ConditionStatus { return c.Status }, Equal(metav1.ConditionTrue)),
```

Reply:
Yikes - just saw this in my local e2e results. It looks like there's another e2e check that has a poor copy-paste job that needs some fixing.